events/api.py

```diff
@@ -718,30 +726,52 @@ class EventSerializer(LinkedEventsSerializer, GeoModelAPIView):
     publication_status = EnumChoiceField(PUBLICATION_STATUSES)
     external_links = EventLinkSerializer(many=True, required=False)
     offers = OfferSerializer(many=True, required=False)
+    id = serializers.CharField(required=False)
```

Comment:
It seems that this allows modifying existing object's ID, which most probably is a bad idea.